addons/web/static/src/views/fields/properties/properties_field.xml

```diff
@@ -11,11 +11,12 @@
                     t-att-property-name="propertyConfiguration.name"
                     t-attf-class="g-col-lg-{{12 / props.columns}} g-col-12">
                     <div class="o_td_label">
-                        <label t-attf-class="o_field_property_label o_form_label {{state.movedPropertyName === propertyConfiguration.name ? 'o_property_field_highlight' : ''}} d-flex flex-row align-items-center">
+                        <label t-attf-class="o_field_property_label o_form_label {{state.movedPropertyName === propertyConfiguration.name ? 'o_property_field_highlight' : ''}} d-flex flex-row align-items-center"
+                            t-attf-for="property-{{propertyConfiguration.name}}">
```

Comment:
Does that mean we are using the name of the property as the DOM ID ?

I seem to remember that we wanted to allow people to specify their own property names, don't we risk collision issues?